app/controllers/api/v2/projects_controller.rb

```diff
@@ -89,8 +87,9 @@ def planning_element_custom_fields
       protected
 
       def find_project
-        @project = Project.find params[:id],
-                                include: [{ custom_values: [{ custom_field: :translations }] }]
+        # FIXME: Rails 4.1 Upgrade: restore includes
+        #        include: [{ custom_values: [{ custom_field: :translations }] }]
+        @project = Project.find params[:id]
```

Comment:
@ulferts seen and commented :) https://github.com/myabc/openproject/commit/51512f15223b3149b409c242c677907dc2047650#commitcomment-12104806